import.go

```diff
@@ -213,7 +213,17 @@ func (i *Importer) Commit() error {
 			len(i.stack))
 	}
 
-	err := i.batch.WriteSync()
+	// Wait for previous batch.
+	var err error
+	if i.inflightCommit != nil {
+		err = <-i.inflightCommit
```

Comment:
maybe using a `sync.Mutex` around batch.Write(Sync)` and `batch.Commit` would be easier to reason about?